bokeh/validation/warnings.py

```diff
@@ -9,16 +9,20 @@
 1002 : *EMPTY_LAYOUT*
     A layout model has no children (will result in a blank layout).
 
+1003 : *COLON_IN_CATEGORY_LABEL*
+    Category label contains colons (will result in a blank layout).
+
```

Comment:
I don't have strong opinions about this, but I do like that `COLON_IN_CATEGORY_LABEL` tells you what the specific problem is. Or are we imagining adding other tests against 1003.